src/coreclr/debug/shared/dbgtransportsession.cpp

```diff
@@ -2499,13 +2499,6 @@ DWORD DbgTransportSession::GetEventSize(DebuggerIPCEvent *pEvent)
         break;
 
     case DB_IPCE_DISABLE_OPTS:
-    case DB_IPCE_IS_OPTS_DISABLED:
```

Comment:
You can also remove `DB_IPCE_IS_OPTS_DISABLED` from `src/coreclr/debug/inc/dbgipceventtypes.h`